subsys/bluetooth/controller/hci/hci.c

```diff
@@ -7002,6 +7002,11 @@ static void le_ext_adv_report(struct pdu_data *pdu_data,
 			ptr += BDADDR_SIZE;
 		}
 
+		if (h->cte_info) {
+			/* CTEInfo is RFU */
+			ptr += 1;
+		}
+
```

Comment:
Is the tester sending extended advertising PDU with CTEInfo? is this a invalid behavior test?

Reply:
It is a test of proper handling of RFU fields being set, yes; The full test name is LL/DDI/SCN/BV-89-C [Extended Scanning, Periodic Advertising, Properly Ignore RFU Fields].

CTEInfo should not be set for an extended advertisement, but if it is it has to be ignored.